tests/queries/0_stateless/02352_lightweight_delete_in_partition.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS t_merge_tree SYNC;
+DROP TABLE IF EXISTS t_replicated_merge_tree SYNC;
+
+CREATE TABLE t_merge_tree(time Date, id String , name String) ENGINE = MergeTree() PARTITION BY time ORDER BY id;
+CREATE TABLE t_replicated_merge_tree(time Date, id String, name String) ENGINE = ReplicatedMergeTree('/test/02352/{database}/t_rep','1') PARTITION BY time ORDER BY id;
+
+INSERT INTO t_merge_tree select '2024-08-01', '1', toString(number) FROM numbers(100);
+INSERT INTO t_merge_tree select '2024-08-02', '2', toString(number)  FROM numbers(100);
+
+INSERT INTO t_replicated_merge_tree select '2024-08-01', '1', toString(number) FROM numbers(100);
+INSERT INTO t_replicated_merge_tree select '2024-08-02', '2', toString(number)  FROM numbers(100);
```

Comment:
you need it having no distinction by `id` in order to test what you intended to test
```suggestion
INSERT INTO t_merge_tree select '2024-08-01', '1', toString(number) FROM numbers(100);
INSERT INTO t_merge_tree select '2024-08-02', '1', toString(number)  FROM numbers(100);

INSERT INTO t_replicated_merge_tree select '2024-08-01', '1', toString(number) FROM numbers(100);
INSERT INTO t_replicated_merge_tree select '2024-08-02', '1', toString(number)  FROM numbers(100);
```